cmd/channel/email/main.go

```diff
@@ -68,6 +68,8 @@ func (ch *Email) Send(reversePath string, recipients []string, msg []byte) error
 
 	if ch.Encryption == EncryptionTLS {
 		client, err = smtp.DialTLS(ch.GetServer(), nil)
+	} else if ch.Encryption == EncryptionStartTLS {
+		client, err = smtp.DialStartTLS(ch.GetServer(), nil)
 	} else {
 		client, err = smtp.Dial(ch.GetServer())
 	}
```

Comment:
Now that all cases can be handled uniformly, I think it would be a good idea to change this to a `switch` statement, make `EncryptionNone` an explicit case and treat everything else as an error instead of silently falling back to the least secure option.